knowledge-base/troubleshoot-report-server.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Troubleshooting Report Server
+description: Diagnose problems with the Telerik Report Server application
+type: troubleshooting
+page_title: Troubleshoot Report Server Manager
+slug: troubleshoot-report-server
+position: 
+tags: 
+ticketid: 1400116
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Report Server</td>
+	</tr>
+</table>
+
+
+## Description
+Sometimes it is possible to experience problems with the Report Server the reasons for which are not apparent. For example, restricted permission for accessing the File Storage, Internal Server Errors due to unaccessible Storage or missing/corrupted report resources (e.g. invalid image), etc.
```

Comment:
unaccessible -> inaccessible / not accessible

Reply:
corrected